packages/web-components/src/components/va-search-input/va-search-input.scss

```diff
@@ -76,7 +76,7 @@
 
 #va-search-button:active,
 #va-search-button:hover {
-  background-color: var(--color-primary-darker);
+  background-color: var(--color-primary-dark);
```

Comment:
Shifts up a level due to names and values being changed per [this comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2228#issuecomment-1836918069)